dataland-backend/src/test/kotlin/org/dataland/datalandbackend/utils/TestDataProvider.kt

```diff
@@ -27,8 +26,8 @@ class TestDataProvider(@Autowired var objectMapper: ObjectMapper) {
             .map { it.copy(identifiers = IdentifierType.values().associateWith { emptyList() }) }
     }
 
-    fun getEmptySmeDataset(): SmeData {
```

Comment:
Why was this changed?

Reply:
Because the function is not required anymore. Where it was used before, we can use a different data type. So there was no need for this one anymore.

Reply:
ok